mbu/forms.py

```diff
@@ -1,6 +1,63 @@
 from django import forms
+from django.forms import ModelForm
+from django.contrib.auth.models import User
+from django.contrib.auth.forms import UserCreationForm
+from django.conf.global_settings import EMAIL_BACKEND
+from mbu.models import Scout, Scoutmaster
+from crispy_forms.helper import FormHelper
+from django.forms.extras.widgets import SelectDateWidget
 
-class EditProfileForm(forms.Form):
-	first_name = forms.CharField(label='First Name')
-	last_name = forms.CharField(label='Last Name')
-	email = forms.CharField(label="Email")
\ No newline at end of file
+class MbuUserCreationForm(UserCreationForm):
```

Comment:
I also don't want them to be able to edit their username without an admin doing it.